lib/packwerk/cli.rb

```diff
@@ -67,7 +69,8 @@ def execute_command(args)
           Subcommands:
             init - set up packwerk
             check - run all checks
-            update-deprecations - update deprecated references
+            todo - update package_todo.yml files
```

Comment:
Personally, I think `update-todo` with a shorthand of `update` would make the most sense. From rubocop's perspective, it is `regenerate-todo`. Since `todo` by itself isn't a verb, I would suggest keeping the update wording in the command.

Reply:
Ah yeah I see your point. I really loved the terseness of `todo` so thought maybe it was worth the tradeoff (of it not being a verb) since it's used so often, but `update` is only two more letters I suppose!! I'll change to `update` and `update-todo`.

Reply:
Naming is hard! 😅  Normally I'd leave a conversation like this untouched as there's rarely value in muddying the waters, but this is the name of a subcommand and therefore pretty important so I thought I'd share my thoughts.

I also appreciate plain `todo` as it is brief and the intentions are clear. Grammatical consistency between subcommands is nice to have but not a hill I'd die on. `update-todo` feels a little unnecessarily wordy to me.

@gmcgibbon thanks for bringing up Rubocop's prior art! I also appreciate consistency and as Packwerk is very similar to Rubocop from the beginner's perspective, copying it is definitely a good thought.

Finally, another option that could work is dropping the `todo` command entirely and just having `packwerk check --todo`. Essentially updating the todo files is just a side effect or mode of a check anyway... 🤔 

My preferences, in order would be:

1. `packwerk check --todo` 
2. `packwerk check --regenerate-todo`
3. `packwerk todo`
4. `packwerk regenerate-todo`
5. `packwerk update-todo`

`packwerk update` feels too vague to me so I think I'd prefer to not support it at all honestly.

Reply:
Personally `todo` doesn't make sense to me because it ins't a verb and may be misleading due to the mutative nature of the command. `todo` also feels wrong because the root of the command is "updating" the todo, so `update` makes sense to me as a partial completion of the full name and the verb of what you're actually doing.

I'm a fan of `packwerk check --update-todo` or `--regenerate-todo` though. Since there's no to type, we can settle on `update` for now and change it later if we find it necessary. If this is all about typing less characters we can always introduce `packwerk u`.

Reply:
Thanks @gmcgibbon and @mclark for both sharing more thoughts. Although I merged this, if @gmcgibbon agrees with being consistent with `tapioca`, I could put up a PR to switch over, given we haven't released anything yet (only merged this PR). Otherwise, we can leave as is.